packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textEditorParts/mask.tests.js

```diff
@@ -1782,6 +1782,32 @@ QUnit.module('clear button', () => {
             clock.restore();
         }
     });
+
+    QUnit.test('Input should has valid value after input text & focusout', function(assert) {
```

Comment:
Let's refactor this test to check the initial scenario.

```suggestion
    QUnit.test('Input text should be cleared after click on a clear button even if value is not changed yet (T1193735)', function(assert) {
```